README.md

```diff
@@ -17,7 +17,7 @@ The official website of the Society is located at [https://hackersatcambridge.co
 
 Any changes to this repository must require a majority vote at a meeting of the Executive Committee. Please view [the template](.github/PULL_REQUEST_TEMPLATE.md) for more information.
 
-Individual files in the repository may relax these requirements for certain files and folders, but these requirements are *only* relaxed for changes which *only* files and folders with relaxed restrictions.
+Individual files in the repository may relax these requirements for certain files and folders, but these requirements are *only* relaxed for those changes which *only* affect files and folders with relaxed restrictions.
```

Comment:
Reads to me like "Requirements are relaxed for files with relaxed restrictions". What are we trying to get at here?

Reply:
I'm trying to say that files in the repo have the power to relax restrictions with regards to making changes to parts of the repo. But that those restrictions are only relaxed for pull-requests which don't touch other files which may still be more heavily restricted.

Reply:
Does that really need to be said? We already say that PRs need to abide by the requirements for the files they are changing

Reply:
OK, but this is an opinionated change to this PR now, so if you want me to go back and change it, I'm happy to do that, but we need to re-vote in the next meeting.

Reply:
Is your main issue that it is worded confusingly?

Reply:
Yes.

Reply:
If this is just reworded to be clearer than I'm happy

Reply:
How about 
“Relaxed restrictions can be specified for individual files or folders. All pull requests must adhere to all restrictions imposed by each file or folder it affects.”

Reply:
👍